NUService.js

```diff
@@ -27,8 +37,11 @@ export default class NUService extends NUObject {
           headerOrderBy: 'X-Nuage-OrderBy',
           headerCount: 'X-Nuage-Count',
           headerMessage:'X-Nuage-Message',
-    }) {
+      }, 
+      URL) {
           super();
+          const url = rootURL ? rootURL : `${URL.protocol}://${URL.hostname}:${URL.port}${URL.RESTRoot}${URL.RESTResource}`
```

Comment:
we talked about parsing the rootURL if available and set host, port, protocol, etc! In NUService set RESTRoot to '/nuage' and RESTResource to '/api' and APIVersion from parsed rootURL